src/questions/questions.controller.ts

```diff
@@ -274,4 +281,102 @@ export class QuestionsController {
       },
     };
   }
+
+  @Get('/:id/invitations')
```

Comment:
You can create a new file and seperate these functions into QuestionsController in that file utilizing the partial definition of Typescript.